src/Microsoft.Health.Fhir.Core/Properties/AssemblyInfo.cs

```diff
@@ -3,17 +3,33 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System.Resources;
 using System.Runtime.CompilerServices;
 
+[assembly: NeutralResourcesLanguage("en-us")]
 [assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
+
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Stu3.Core")]
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.R4.Core")]
+[assembly: InternalsVisibleTo("Microsoft.Health.Fhir.R5.Core")]
+
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Stu3.Core.UnitTests")]
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.R4.Core.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Health.Fhir.R5.Core.UnitTests")]
+
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.CosmosDb")]
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.SqlServer")]
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Api")]
+
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Api.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Health.Fhir.R4.Api.UnitTests")]
```

Comment:
super nit: would be nice to have same Stu3-R3-R4 ordering of assemblies within this file :)